praw/models/helpers.py

```diff
@@ -94,16 +94,29 @@ def __call__(self, display_name):
         return Subreddit(self._reddit, display_name=display_name)
 
     def create(self, name, link_type, subreddit_type, title, wikimode,
-               **kwargs):
+               **other_settings):
         """
         Create a new subreddit.
 
-        See :meth:`~Subreddit._create_or_update` for details on each parameter.
+        :param link_type: The types of submissions users can make.
+            One of ``any``, ``link``, ``self``.
+        :param name: The name for the new subreddit.
+        :param subreddit_type: One of ``archived``, ``employees_only``,
+            ``gold_only``, ``gold_restricted``, ``private``, ``public``,
+            ``restricted``.
+        :param title: The title of the subreddit.
+        :param wikimode: One of  ``anyone``, ``disabled``, ``modonly``.
+
+        See :meth:`~.SubredditModeration.update` for documenation on other
```

Comment:
Typo `documentation`. Could we say, "See .. for documentation of other available settings.